examples/basic-cn/tidb-monitor.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: pingcap.com/v1alpha1
+kind: TidbMonitor
+metadata:
+  name: basic
+spec:
+  clusters:
+  - name: basic
+  prometheus:
+    baseImage: prom/prometheus
```

Comment:
Should we change the images of prom and grafana to the ones in `registry.cn-beijing.aliyuncs.com/tidb` too?

Reply:
I suggest not, because it will break once users change Prometheus version unless we sync all Prometheus images to our registry

tidb monitor is not optional